app/main.py

```diff
@@ -1,8 +1,49 @@
+from typing import List
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
```

Comment:
```suggestion
    def __init__(
            self,
            comfort_class: int,
            clean_mark: int,
            brand: str
    ) -> None:
```

Reply:
you should move each parameter on a new line. Fix in all places

Reply:
so I have each parameter in a new line, or I don't understand something

Reply:
Your first choice is also legit; however, when you use type annotation and, especially in the `__init__`, it is conventional to place each argument on a new line and leave opening and closing parenthesis without arguments on the same line.